DefaultRoutine/Chuck.SilverFish/cards/04Expansion/009BOT/Sim_BOT_299.cs

```diff
@@ -24,13 +24,13 @@ public override void onCardPlay(Playfield p, bool ownplay, Minion target, int ch
             {
                 p.drawACard(CardName.unknown, ownplay, true);
                 p.drawACard(CardName.unknown, ownplay, true);
+                p.drawACard(CardName.unknown, ownplay, true);
             }
 
-            if (!ownplay && p.enemyMaxMana == 10)
+            if (ownplay && !p.enemyMaxMana == 10)
```

Comment:
Please learn basic knowledge about C# type bool, and install visual studio 2019.